pkg/testers/clusterloader2/cl2.go

```diff
@@ -44,6 +45,7 @@ type Tester struct {
 	Nodes                     int    `desc:"Number of nodes in the cluster. 0 will auto-detect schedulable nodes."`
 	EnablePrometheusServer    bool   `desc:"Whether to set-up the prometheus server in the cluster."`
 	PrometheusPvcStorageClass string `desc:"Storage class used with prometheus persistent volume claim."`
+	ExtraCL2Args              string `flag:"~extra-cl2-args" desc:"Additional arguments supported by clusterloader2 (https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/cmd/clusterloader.go)."`
```

Comment:
can we leave the flag as just `extra-args`?

Reply:
done